Makefile

```diff
@@ -684,6 +684,7 @@ test-browser:
 		-v ${OPSTRACE_BUILD_DIR}/browser-test-results:/build/test/browser/test-results \
 		-e OPSTRACE_CLUSTER_NAME \
 		-e OPSTRACE_CLOUD_PROVIDER \
+		-e OPSTRACE_AUTH_METHOD=test \
```

Comment:
Let's be deliberate about differentiating between "login" and "authentication".

* "login" is exchanging primary credentials into secondary credentials (a single sign-on, for example, or a username/password login in the classical sense). This is a _rare_ event compared to "authentication".
* "authentication" usually refers to decorating an HTTP request with authentication proof where the secondary credentials (like an authentication token) is _added_ to the request. This is a _frequent_ event compared to "login".

As of having dealt with this in the past years it really really really makes sense to get aligned on this terminology :-), especially in code and then also issues/bug reports.

Reply:
(if this is actually LOGIN_METHOD... let's rename :))